stable/aws-efa-k8s-device-plugin/Chart.yaml

```diff
@@ -0,0 +1,9 @@
+apiVersion: v1
+name: aws-efa-k8s-device-plugin
+description: A Helm chart for EFA device plugin.
+version: 0.1.0
```

Comment:
last nit: this should be the same version as `appVersion`, so `v0.4.2` currently